be/src/util/simd/vstring_function.h

```diff
@@ -187,6 +187,19 @@ class VStringFunctions {
         return p;
     }
 
+    static inline std::pair<size_t, size_t> skip_leading_utf8(const char* begin, const char* end,
```

Comment:
Add comment and change the func name. we should know what the functinon do. if just one place call the func, better be a lambda